basics/a51.go

```diff
@@ -0,0 +1,18 @@
+package main
+import "fmt"
+
+type Celsius float64
+type Fahrenheit float64
+
```

Comment:
golint is comlaining:

a51.go:4:6: exported type Celsius should have comment or be unexported
a51.go:5:6: exported type Fahrenheit should have comment or be unexported

Reply:
Updated the code.